process-compose/suzuka-full-node/process-compose.yml

```diff
@@ -40,15 +40,6 @@ processes:
       initial_delay_seconds: 3
       exec:
         command: grpcurl -plaintext 0.0.0.0:30730 list
```

Comment:
Goodbye

Reply:
I don't see why the test of the grpc port of the lightnode should be removed?

Reply:
Below it's the suzuka-full-node entry. I still don't see. 
I see there's a postgres enry in the suzuka-full-node/docker-compose.yml. Postgres is not needed for the Suzuka-full. Perhaps it should be removed and added to the indexer docker-compose or just removed. Depends on how postgres db is managed.